notifier_types.go

```diff
@@ -78,6 +80,10 @@ func (not *Notifier) GetSendAllEvents() bool {
 	return *not.SendAllEvents
 }
 
+func (not *Notifier) GetSendAllAlerts() bool {
+	return *not.SendAllAlerts
```

Comment:
What happens if `SendAllAlerts` is nil?

Reply:
It's not.

Actually this is only used when data is read from the API to specialize the different types, the boolean value is always defined.

It's a pointer because when send as parameter to the API for Create/Update, we want the client to be able to ignore the parameter and not send a meaningful value like `true` or `false`